Sources/App/configure.swift

```diff
@@ -334,6 +334,11 @@ public func configure(_ app: Application) throws -> String {
     app.commands.use(TriggerBuildsCommand(), as: "trigger-builds")
     app.commands.use(ReAnalyzeVersions.Command(), as: "re-analyze-versions")
 
+#warning("move to proper place before merging")
+    do {  // Migration 0xx - Add `build_errors` to `builds`
+        app.migrations.add(UpdateBuildAddBuildErrors())
+    }
+
```

Comment:
This is out of order only so it doesn't conflict with the migration in branch `fix-2970`. Will move to proper sequence before merging, depending on which one ends up going first.

Reply:
Updated